packages/astro/components/index.ts

```diff
@@ -4,5 +4,3 @@
 export { default as Code } from './Code.astro';
 // @ts-ignore
 export { default as Debug } from './Debug.astro';
-// @ts-ignore
-export { default as Welcome } from './Welcome.astro';
```

Comment:
It moved it back into the template for multiple reasons:

- You can't import from `astro:components` a component that has styles defined the "Astro way", you cause styles to leak everywhere
- Inside Astro, you could check out the code, but not modify it, which feels like pedagogically it was a downgrade

Reply:
FWIW (and perhaps you already know) for no1, I think `is:inline` can be used to avoid that